dmriprep/cli/run.py

```diff
@@ -416,7 +418,11 @@ def build_workflow(opts, retval):
         retval['return_code'] = 1
         return retval
 
-    output_spaces = parse_spaces(opts)
+    # Initialize --output-spaces if not defined
+    output_spaces = opts.output_spaces
+    if opts.output_spaces is None:
+        from niworkflows.utils.spaces import Reference, SpatialReferences
+        output_spaces = SpatialReferences([Reference("orig", {})])
```

Comment:
"orig" is not a recognized reference

Reply:
shoot yes, will change

Reply:
mmm I'll go with `run` but there's no good option, really: https://github.com/nipreps/niworkflows/blob/master/niworkflows/utils/spaces.py#L8-L17

I'll open an issue for this.